config/samples/v2/external-object-storage/dspa.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: datasciencepipelinesapplications.opendatahub.io/v1alpha1
+kind: DataSciencePipelinesApplication
+metadata:
+  name: sample
+spec:
+  dspVersion: v2
+  apiServer:
+    image: gcr.io/ml-pipeline/api-server:2.0.2
+  persistenceAgent:
+    image: gcr.io/ml-pipeline/persistenceagent:2.0.2
+  scheduledWorkflow:
+    image: gcr.io/ml-pipeline/scheduledworkflow:2.0.2
+  mlmd:
+    grpc:
+      image: gcr.io/tfx-oss-public/ml_metadata_store_server:1.14.0
+    envoy:
+      image: gcr.io/ml-pipeline/metadata-envoy:2.0.2
+  objectStorage:
+    externalStorage:
+      bucket: mybucket
+      host: s3.amazonaws.com
```

Comment:
can you add the region here like [this](https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/samples/external-object-storage/dspa.yaml#L9)

Reply:
Done.